### PR TITLE
NW6 | Fathi Kahin | React-Module-Project |SearchResults-WEEK-1

### PR DESCRIPTION
**Pull request by fhkahin**
## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/componentsSearchResults/componentsSearchResults.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import TableBody from './TableBody';
+
+const SearchResults = () => {
+  return (
+    <table>
+      <thead>
+        <TableHead />
+      </thead>
+      <tbody>
+        {FakeBookings.map((booking) => (
```
Comment (RbAvci) [inline src/components/componentsSearchResults/componentsSearchResults.jsx:14]: The search results should be passed as props to this component. If you can check `<Bookings />` component in `Bookings/Bookings.jsx`, you can see the `FakeBookings` data is passed to `SearchResults` component. See here: https://github.com/RbAvci/React-Module-Project/blob/bbbed68ba6a9d4a0e19be7c916785bf2b0712ab0/src/components/Bookings/Bookings.jsx#L13.

**Review by RbAvci — DISMISSED** [review]
Great work Fathi! However some changes needed:
- The folder and file name (`componentsSearchResults`) should be `SearchResults`. 
- Component is not rendered in the page because it is commented in `<Boookings/>` component. You should uncomment it.
- Tests need to be implemented as well.

File: src/components/SearchResults/SearchResults.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const SearchResults = () => {
+  return (
+    <table>
+      <thead>
+        <TableHead />
+      </thead>
+      <tbody>
+        {FakeBookings.map((booking) => (
+          <TableBody
+            key={booking.id}
```
Comment (fikretellek) [inline src/components/SearchResults/SearchResults.jsx:16]: good practice using key property

File: src/components/SearchResults/TableBody.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const TableBody = ({ id, title, firstName, surName, email }) => {
+  return (
+    <tr>
```
Comment (fikretellek) [inline src/components/SearchResults/TableBody.jsx:5]: table elements look great

**Review by fikretellek — APPROVED** [review]
Great work. Well done!!!

**Comment by Ara225** [comment]
@RbAvci Doing a great job of reviewing :)

**Pull request merged**